libraries/networking/src/udt/Connection.cpp

```diff
@@ -0,0 +1,597 @@
+//
+//  Connection.cpp
+//  libraries/networking/src/udt
+//
+//  Created by Clement on 7/27/15.
+//  Copyright 2015 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+#include "Connection.h"
+
+#include <QtCore/QThread>
+
+#include <NumericalConstants.h>
+
+#include "../HifiSockAddr.h"
+#include "CongestionControl.h"
+#include "ControlPacket.h"
+#include "Packet.h"
+#include "Socket.h"
+
+using namespace udt;
+using namespace std;
+using namespace std::chrono;
```

Comment:
I'd side with you on the std side, but for std::chrono it can make for many broken lines which makes what we're doing a lot more difficult to read.